aepp/flowservice.py

```diff
@@ -1306,7 +1324,30 @@ def getLandingZoneContainerTTL(
             dlz_type : OPTIONAL : The type of DLZ container - default to "user_drop_zone" but can be "dlz_destination"
         """
         return int(self.getLandingZoneContainer(dlz_type=dlz_type)["containerTTL"])
-    
+
+    def getLandingZoneStorageTTL(
+            self,
+            dlz_type: str = "user_drop_zone"
+    ) -> int:
+        """
+        Returns the TTL in days of the DLZ storage corresponding to this type.
+        Arguments:
+            dlz_type : OPTIONAL : The type of DLZ container - default to "user_drop_zone" but can be "dlz_destination"
+        """
+        response = self.getLandingZoneContainer(dlz_type=dlz_type)
+        if self._isAzureLandingZone(response):
+            return int(response["containerTTL"])
+        else:
+            return int(response["TTL"])
+
+    def _isAzureLandingZone(self, landingZoneResponse) -> bool:
+        """
+        Inspects landingZoneResponse structure, if "containerName" - is present this flags Azure otherwise its AWS;
+        Arguments:
+            landingZone : REQUIRED : response object of landingzone or landingzone/credentials invocation
+        """
+        return 'containerName' in landingZoneResponse
```

Comment:
I think it would be more robust to use `dlzProvider` that will be available in every DLZ response moving forward.

Reply:
right, however as or right now dlzProvider is not part of landingzone response body (i haven't received confirmation whether it will be introduced tho) 'containerName' -seem to be present in all GET calls, i'll keep it as is for now, I'll delay this PR until DLZ changes are in AEP.

Reply:
@cmenguy - leaving existing impl, for azure case the response for getLandingZoneContainer() haven't changed 
and stays to this day for azure:
```
{'containerName': 'dlz-user-container', 'containerTTL': '7'}
```
while for aws it returns:

```
{'dlzPath': {'bucketName': 'dlz-prod-65da01ac-a93f-4b3b-9a01-aca93fcb3b07', 'dlzFolder': 'dlz-user-container'}, 'dataTTL': {'timeUnit': 'days', 'timeQuantity': 7}, 'dlzProvider': 'Amazon S3'}
```